src/vue/components/info.vue

```diff
@@ -40,38 +41,42 @@
 import DependencyCheck from '@/vue/vuex-modules/support/dependency-check.js'
 import ReadingToolsIcon from '@/images/inline-icons/reading-tools.svg'
 import LookupIcon from '@/images/inline-icons/lookup.svg'
-import OptionsIcon from '@/images/inline-icons/options.svg'
 import LogoIcon from '@/images/alpheios/logo.svg'
 import SwapPosition from '@/images/inline-icons/swap-horizontally.svg'
 import TapGestureIcon from '@/images/inline-icons/tap-gesture-icon.svg'
 
 export default {
   name: 'Info',
-  inject: ['app', 'l10n'],
+  inject: ['app', 'l10n', 'lexis'],
   storeModules: ['app'],
   components: {
-    optionsIcon: OptionsIcon,
     readingToolsIcon: ReadingToolsIcon,
     lookupIcon: LookupIcon,
     logoIcon: LogoIcon,
     swapPosition: SwapPosition,
     tapGestureIcon: TapGestureIcon
   },
   mixins: [DependencyCheck],
+
+  data: () => {
+    return {
+      cedictResponse: ''
+    }
+  },
```

Comment:
what is the reason for this change ?

Reply:
This is a leftover from a test code that was showing results of a CEDICT request in an `info` component. This is not needed now and I will remove it. Thanks for a good catch!